.env

```diff
@@ -0,0 +1,3 @@
+VITE_SOLID_IDENTITY_PROVIDER_PRODUCTION="https://opencommons.net"
+
+VITE_SOLID_IDENTITY_PROVIDER_DEV="http://localhost:3000"
```

Comment:
Nice! Thanks for starting a branch for this issue and make a PR.

With regards to an alternative Solid Identity Provider or OIDC issue, I think we should use solidcommunity.net instead of localhost:3000.

This way PASS is set up to a valid Solid server.

Reply:
The developer can certainly redirect PASS to point at a different Identity Provider. They need simply change these values to a different URL. I just wanted to include some sensible defaults here. And move the URLs to a more standard file. I actually think it's best to remove this `.env` file from source control entirely after distributing it.

I think it's a good idea to run against localhost at least while developing since `solidcommunity.net` can be taken down at any time. If that happens, our development will stall. If we are instead able to run a server on our own computer, each developer will be able to put up their own server and keep developing normally. I'm just trying to make that easier.

Reply:
Perhaps I'm a bit unclear as to the changes being made. But, I'm assuming it suppose to allow us to bypass login and allow us to go to different pages? Let me know if this is not correct.

Of course, we can't onto a Solid Pod without logging in, so I can ignore that for now. However, I can't seem to access /PASS/home/ and /PASS/forms/ using localhost:3000.

Reply:
Ah, never mind. I understand now. I was a misunderstanding on my part. This would be alright with me as a fix/potential start to migrating to Community Solid Server, albeit locally for now.